ansible/roles/ocp-workload-vertx-reactica/tasks/main.yml

```diff
@@ -1,22 +1,20 @@
 ---
-
 - name: Running Pre Workload Tasks
-  include: ./pre_workload.yml
-  become: "{{ become_override | bool }}"
+  import_tasks: ./pre_workload.yml
+  become: false
   when: ACTION == "create" or ACTION == "provision"
 
 - name: Running Workload Tasks
-  include: ./workload.yml
-  become: "{{ become_override | bool }}"
+  import_tasks: ./workload.yml
+  become: false
   when: ACTION == "create" or ACTION == "provision"
 
-
 - name: Running Post Workload Tasks
-  include: ./post_workload.yml
-  become: "{{ become_override | bool }}"
```

Comment:
please keep `become_override`. if one day we want to use this workload while deploying a cluster, we will `include_role` that role (applying it to master1) and set `become_override` to true

Reply:
Ok. Fixed